### ISSUE TITLE
play custom sound on camera speaker

### ISSUE THREAD

**Issue by aaronlower** [briis/unifiprotect#304]
feature request to play a custom sound on the camera speaker (e.g. G3 instant, G4 dome, G4 doorbell) when person is detected or another event is triggered

**Comment by AngellusMortis** [comment]
I can confirm I have figured out the `talkbackSettings` and I am implementing an interface for this in `pyunifiprotect` so we can hopefully implement a `media_player` entity for cameras that have speakers / talkback in the "near ish" future (maybe for 0.11?).

**Comment by AngellusMortis** [comment]
Added to the 0.11 beta. Doorbells may still have some issues that I am looking into more, but those may be the result of general doorbell issues everyone else has had post 1.20 as well.

**Comment by disabledtech** [comment]
Hi,

I'm having some issues with the media_player for my G4 doorbell. I can get it to play TTS messages and custom sounds (e.g. a buzzer .mp3) but only a few times before it stops working. To get it playing again I need to reboot the doorbell. After a reboot I can do a few more sound/TTS before it stops working again.

Home Assistant version: 2021.12.3 running on a raspberry pi 4B 8GB
UniFi Protect Integration: 0.11.4 via HACS
UniFi Protect: 1.20.3 running on a UDM Pro
G4 Firmware: 4.46.18

I enable debugging in my config. I see this line every 2 seconds.

`2021-12-29 17:28:36 DEBUG (MainThread) [pyunifiprotect.api] Skipping update since websocket is active`

Here's a longer snippet of the logs directly after trying to use TTS directly through the G4 Doorbell Speaker entity.

```
2021-12-29 17:32:30 DEBUG (MainThread) [pyunifiprotect.api] Skipping update since websocket is active
2021-12-29 17:32:32 DEBUG (MainThread) [pyunifiprotect.api] Skipping update since websocket is active
2021-12-29 17:32:33 DEBUG (MainThread) [custom_components.unifiprotect.media_player] Playing Media http://192.168.1.119:8123/api/tts_proxy/a94a8fe5ccb19ba61c4c0873d391e987982fbbd3_en-us_826d21f2a0_cloud.mp3 for UVC G4 Doorbell Speaker
2021-12-29 17:32:33 DEBUG (MainThread) [pyunifiprotect.stream] ffmpeg: /usr/bin/ffmpeg -loglevel info -hide_banner -i http://192.168.1.119:8123/api/tts_proxy/a94a8fe5ccb19ba61c4c0873d391e987982fbbd3_en-us_826d21f2a0_cloud.mp3 -vn -acodec aac -ac 1 -ar 22050 -b:a 48000 -map 0:a -f adts udp://192.168.1.222:7004
2021-12-29 17:32:33 DEBUG (MainThread) [pyunifiprotect.data.devices] ffmpeg stdout:

2021-12-29 17:32:33 DEBUG (MainThread) [pyunifiprotect.data.devices] ffmpeg stderr:
[mp3 @ 0x7f9e275000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/379bddf814a27148dde6bd081a5e0534f9397925_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : en-US
  Duration: 00:00:03.41, start: 0.000000, bitrate: 50 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : en-US
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      22kB time=00:00:03.39 bitrate=  52.3kbits/s speed=45.3x
video:0kB audio:21kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.426624%
[aac @ 0x7f9e2a7e00] Qavg: 19856.703
[mp3 @ 0x7f9f875000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/daaea13eb2f37c90638b8469f55e9eacb0622f5a_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : en-US
  Duration: 00:00:03.48, start: 0.000000, bitrate: 50 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : en-US
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
[mp3float @ 0x7f9f8a6f00] overread, skip -5 enddists: -3 -3
size=      22kB time=00:00:03.43 bitrate=  51.4kbits/s speed=47.4x
video:0kB audio:21kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.469824%
[aac @ 0x7f9f8a7e00] Qavg: 17516.441
[mp3 @ 0x7fb8275000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/a94a8fe5ccb19ba61c4c0873d391e987982fbbd3_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : test
  Duration: 00:00:01.51, start: 0.000000, bitrate: 53 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : test
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=       6kB time=00:00:01.48 bitrate=  34.9kbits/s speed=33.5x
video:0kB audio:6kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 3.811659%
[aac @ 0x7fb82a7e00] Qavg: 37139.648
[mp3 @ 0x7fbaa75000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/a94a8fe5ccb19ba61c4c0873d391e987982fbbd3_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : test
  Duration: 00:00:01.51, start: 0.000000, bitrate: 53 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : test
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=       6kB time=00:00:01.48 bitrate=  34.9kbits/s speed=34.9x
video:0kB audio:6kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 3.811659%
[aac @ 0x7fbaaa7e00] Qavg: 37139.648
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=  49x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7f7dccbe00] Qavg: 1634.159
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=49.5x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7fa3ecbe00] Qavg: 1634.159
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=47.1x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7fad6cbe00] Qavg: 1634.159
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=39.3x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7fa2ccbe00] Qavg: 1634.159
[mp3 @ 0x7f88c75000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/a94a8fe5ccb19ba61c4c0873d391e987982fbbd3_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : test
  Duration: 00:00:01.51, start: 0.000000, bitrate: 53 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : test
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=       6kB time=00:00:01.48 bitrate=  34.9kbits/s speed=  35x
video:0kB audio:6kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 3.811659%
[aac @ 0x7f88ca7e00] Qavg: 37139.648
[mp3 @ 0x7f8a075000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/daaea13eb2f37c90638b8469f55e9eacb0622f5a_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : en-US
  Duration: 00:00:03.48, start: 0.000000, bitrate: 50 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : en-US
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
[mp3float @ 0x7f8a0a6f00] overread, skip -5 enddists: -3 -3
size=      22kB time=00:00:03.43 bitrate=  51.4kbits/s speed=  45x
video:0kB audio:21kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.469824%
[aac @ 0x7f8a0a7e00] Qavg: 17516.441
[mp3 @ 0x7fa8475000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/daaea13eb2f37c90638b8469f55e9eacb0622f5a_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : en-US
  Duration: 00:00:03.48, start: 0.000000, bitrate: 50 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : en-US
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
[mp3float @ 0x7fa84a6f00] overread, skip -5 enddists: -3 -3
size=      22kB time=00:00:03.43 bitrate=  51.4kbits/s speed=46.9x
video:0kB audio:21kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.469824%
[aac @ 0x7fa84a7e00] Qavg: 17516.441
[mp3 @ 0x7fb6a75000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/379bddf814a27148dde6bd081a5e0534f9397925_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : en-US
  Duration: 00:00:03.41, start: 0.000000, bitrate: 50 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : en-US
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=N/A    
size=      22kB time=00:00:03.39 bitrate=  52.3kbits/s speed=43.6x
video:0kB audio:21kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.426624%
[aac @ 0x7fb6aa7e00] Qavg: 19856.703
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=46.4x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7f986cbe00] Qavg: 1634.159
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=39.6x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7fad0cbe00] Qavg: 1634.159
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=36.4x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7f8eccbe00] Qavg: 1634.159
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=49.9x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7f9facbe00] Qavg: 1634.159
Input #0, mp3, from 'http://homeassistant:8123/local/sfx/buzzer.mp3':
  Metadata:
    encoder         : Lavf58.76.100
  Duration: 00:00:01.96, start: 0.025057, bitrate: 153 kb/s
  Stream #0:0: Audio: mp3, 44100 Hz, mono, fltp, 153 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
size=      12kB time=00:00:01.90 bitrate=  51.0kbits/s speed=47.6x
video:0kB audio:12kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.544594%
[aac @ 0x7f928cbe00] Qavg: 1634.159
[mp3 @ 0x7fab675000] Estimating duration from bitrate, this may be inaccurate
Input #0, mp3, from 'http://192.168.1.119:8123/api/tts_proxy/a94a8fe5ccb19ba61c4c0873d391e987982fbbd3_en-us_826d21f2a0_cloud.mp3':
  Metadata:
    Text            : test
  Duration: 00:00:01.51, start: 0.000000, bitrate: 53 kb/s
  Stream #0:0: Audio: mp3, 24000 Hz, mono, fltp, 48 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (mp3 (mp3float) -> aac (native))
Press [q] to stop, [?] for help
Output #0, adts, to 'udp://192.168.1.222:7004':
  Metadata:
    Text            : test
    encoder         : Lavf58.76.100
  Stream #0:0: Audio: aac (LC), 22050 Hz, mono, fltp, 48 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
size=       0kB time=00:00:00.00 bitrate=N/A speed=N/A    
size=       6kB time=00:00:01.48 bitrate=  34.9kbits/s speed=35.2x
video:0kB audio:6kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 3.811659%
[aac @ 0x7fab6a7e00] Qavg: 37139.648
2021-12-29 17:32:34 DEBUG (MainThread) [pyunifiprotect.api] Skipping update since websocket is active
2021-12-29 17:32:36 DEBUG (MainThread) [pyunifiprotect.api] Skipping update since websocket is active
```

**Comment by AngellusMortis** [comment]
Probably fixed by https://github.com/briis/unifiprotect/pull/438. Please wait until that gets merged and 0.12.0-beta2 is released. If the issue persists, please create a new issue.
